test/suites/integration/bsp/debt-collection.test.ts

```diff
@@ -301,20 +312,24 @@ describeBspNet(
       await userApi.sealBlock();
 
       // Get the current price of storage from the runtime, the new stream deposit and the ED
-      const currentPriceOfStorage = await userApi.query.paymentStreams.currentPricePerUnitPerTick();
+      const currentPriceOfStorage =
+        await userApi.query.paymentStreams.currentPricePerGigaUnitPerTick();
       const newStreamDeposit = userApi.consts.paymentStreams.newStreamDeposit;
       const existentialDeposit = userApi.consts.balances.existentialDeposit;
 
       // Get the current free balance of the user
       const freeBalance = (await userApi.query.system.account(userAddress)).data.free;
 
       // To make the user insolvent, we need to update the payment stream with a very high amount
-      // and advance new stream deposit blocks
-      // To do this, the new amount provided should be equal to the free balance of the user divided by
-      // the current price of storage multiplied by the new stream deposit
+      // of amount provided so when the BSP tries to charge it the user cannot pay its debt.
+      // We set the new provided amount to be as much as possible considering the deposit
+      // that the user is going to have to pay. We leave 10 * ED in the account to allow
+      // the user to pay its other debts.
```

Comment:
```suggestion
      // so that when the BSP attempts to charge it, the user cannot cover the debt.
      // We set the new provided amount to be as much as possible, considering the deposit
      // that the user is required to pay. We leave 10 * ED in the account to ensure
      // the user can still pay other debts.
```